synapse/cortex.py

```diff
@@ -854,6 +854,21 @@ class Cortex(s_oauth.OAuthMixin, s_cell.Cell):  # type: ignore
             'description': 'Enable Storm scrape interfaces when using $lib.scrape APIs.',
             'type': 'boolean',
         },
+        'storm:query:mirror': {
+            'description': 'Configure a Cortex mirror or AHA pool to offload Storm queries to. This is an experimental option and should not be used in production.',
+            'type': 'object',
```

Comment:
Can we break these into two separate knobs?

Reply:
Yeah I couldn't decide whether this should be separate options or one conf object, I'll split it.